azure-pipelines.yml

```diff
@@ -61,6 +67,22 @@ jobs:
       platform: x64
       graphics_api: D3D12
 
+  - template: .github/jobs/win32.yml
+    parameters:
+      name: Win32_x64_D3D12_Sanitizers
+      vmImage: 'windows-latest'
+      platform: x64
+      graphics_api: D3D12
+      enableSanitizers: true
+
+  - template: .github/jobs/win32.yml
+    parameters:
+      name: Win32_x64_V8_D3D11_Sanitizers
+      vmImage: 'windows-latest'
+      platform: x64
+      napiType: V8
+      enableSanitizers: true
```

Comment:
How much time does this add to the CI if any? Do we need to run both with sanitizers and without?